app/internal/quotes/load_quotes.py

```diff
@@ -1,41 +0,0 @@
-import json
```

Comment:
You should remove this file using `git rm`

Reply:
I think I've already removed these files.. I did git rm before the first commit and now again after the pull upstream. I dont have these files in my updated branch...
Maybe I missed something?

Reply:
Maybe `git rm --cached [full_file_path]`.

Remember to `git commit` and `git push` after running `git rm`.

Reply:
Yes but I did it.. I don't understand, the deleted files are no longer in my branch and in the PR it is written that they are deleted..

Reply:
In addition to my previous comment, I want to make sure we are aligned - both of the deleted files exist in the develop branch(already merged). I replaced them as I refactored the code. Maybe that's why they appear in the pr.
And as I understand, they will be removed just after the merge back to develop...